src/main/java/teammates/ui/controller/FeedbackSubmissionEditSaveAction.java

```diff
@@ -296,6 +298,21 @@ protected void checkAdditionalConstraints() {
         return;
     }
     
+    /**
+     * Note that when overriding this method, this should not use {@code respondingStudentList} 
+     * or {@code respondingInstructorList} of {@code FeedbackSessionAttributes}.
```

Comment:
Add an explanation to this comment so it is easier to understand why?
Other than that, looks good to merge.